cli/Cargo.toml

```diff
@@ -4,6 +4,10 @@ version = "0.9.0"
 authors = ["Taehoon Moon <taehoon.moon@outlook.com>"]
 edition = "2021"
 
+[[bin]]
+name = "gluesql"
+path = "src/main.rs"
+
```

Comment:
@ever0de Oh..! Proper place where `[[bin]]` needs to be located was `cli/Cargo.toml`! not `Cargo.toml` in root.